<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cpp_qt_ia_main_winClass</class>
 <widget class="QMainWindow" name="cpp_qt_ia_main_winClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>913</width>
    <height>626</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>cpp_qt_ia_main_win</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>Settings</string>
        </property>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>20</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Input A</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="edInputA">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>40</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>10</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>100</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Input B</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="edInputB">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>120</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>15</string>
         </property>
        </widget>
        <widget class="QPushButton" name="butCalculate">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>150</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Calculate</string>
         </property>
        </widget>
        <widget class="QComboBox" name="cbOperation">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>70</y>
           <width>111</width>
           <height>22</height>
          </rect>
         </property>
         <item>
          <property name="text">
           <string>+</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>-</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>*</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>/</string>
          </property>
         </item>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Output</string>
        </property>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>20</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Result</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="edOutput">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>40</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
        <widget class="QPlainTextEdit" name="memoResults">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>70</y>
           <width>161</width>
           <height>261</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="butAverage">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>340</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Calc Average</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="edAverage">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>340</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>913</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="cpp_qt_ia_main_win.qrc"/>
 </resources>
 <connections/>
</ui>
